### ISSUE TITLE
[Code] Training loss

### ISSUE THREAD

**Issue by wj-data** [NIRVANALAN/STream3R#7]
Before public the training code, could you mind providing the code of training loss? Thank you very much.

**Comment by YibinWu** [comment]
Hi @Luo-Yihang , any update on the training code? Thanks.

**Comment by NIRVANALAN** [comment]
Hi, we are polishing the training code and will first release the training loss file this week. Will keep you updated.

**Comment by Luo-Yihang** [comment]
Hi, we’ve updated the training loss code in the `train` branch. The full training code will be released in the coming month. Thanks for your patience!

https://github.com/NIRVANALAN/STream3R/blob/5e4d2dd01d3e09593b8626aafc6ed2b89dddbb9e/stream3r/loss/losses.py#L227

**Comment by wj-data** [comment]
@Luo-Yihang thanks for your impressive and helpful work!!
